forms-flow-bpm/src/main/resources/processes/telework-form.bpmn

```diff
@@ -73,40 +74,77 @@ task.execution.setVariable('deleteReason', "completed");</camunda:script>
         </camunda:connector>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_0ociprs</bpmn:incoming>
+    </bpmn:serviceTask>
+    <bpmn:sequenceFlow id="Flow_0eqfxon" sourceRef="StartEvent_1" targetRef="Activity_13ovu4t">
+      <bpmn:extensionElements>
+        <camunda:executionListener class="org.camunda.bpm.extension.hooks.listeners.FormBPMDataPipelineListener" event="take" />
+      </bpmn:extensionElements>
+    </bpmn:sequenceFlow>
+    <bpmn:serviceTask id="Activity_13ovu4t" name="Send Approval Email to Supervisor">
+      <bpmn:extensionElements>
+        <camunda:connector>
+          <camunda:inputOutput>
+            <camunda:inputParameter name="subject">FOR ACTION: A new Telework Agreement has been submitted</camunda:inputParameter>
+            <camunda:inputParameter name="to">${managerEmployerEMail}</camunda:inputParameter>
```

Comment:
Thoughts on updating this to `managerEmail`? Or something a bit simpler as we'll have to make sure it aligns with the auto-population work
CC/ @iman-jamali-fw

Reply:
Sure, that works.  I just used what was in place on the current iteration of the telework form on dev, but that does make sense.